magefiles/lint.go

```diff
@@ -0,0 +1,98 @@
+//go:build mage
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/sh"
+)
+
+type Lint mg.Namespace
+
+// Run all linters
+func (l Lint) All() error {
+	mg.Deps(l.Go, l.Extra)
+	return nil
+}
+
+// Lint everything that's not code
+func (l Lint) Extra() error {
```

Comment:
maybe `NonCode`?

Reply:
I based it off of the GHA name, but don't have a strong preference either way